src/app/services/cartService.ts

```diff
@@ -0,0 +1,34 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from "@angular/core";
+import { ProductIdQuantity } from "../entities/productIdQuantity";
+
+
+@Injectable({
+    providedIn: 'root'
+})
+
+export class CartService {
+
+    URL = "http://localhost:3000/products";
```

Comment:
**suggestion:** you can have the url inside the `environment.ts` file or you can use a [proxy](https://angular.io/guide/build#proxying-to-a-backend-server)